Firmware/ultralcd.cpp

```diff
@@ -8968,6 +8968,7 @@ void lcd_ignore_click(bool b)
 }
 
 void lcd_finishstatus() {
+  SERIAL_PROTOCOLLN("LCD status changed");
```

Comment:
Please change `SERIAL_PROTOCOLLN()` to `SERIAL_PROTOCOLLNPGM()`

Reply:
ok, but why?

Reply:
Also, please think of a more abbreviated string. Take this as an example:
`EEPROM value changed` -> `EEPROMChng`

Reply:
The PGM attribute puts the string in a different memory space (PROGMEM) instead of RAM directly. As a result, it will use less of the already limited RAM at the cost of a few more instructions

Reply:
I talked about it with Tomáš and we not sow abbreviations in other messages from fw. I'm for abbreviations but we must declare some system of abbreviations.